packages/integration-tests/suites/stacktraces/protocol_containing_fn_identifiers/test.ts

```diff
@@ -0,0 +1,55 @@
+import { expect } from '@playwright/test';
+
+import { sentryTest } from '../../../utils/fixtures';
+import { getSentryRequest } from '../../../utils/helpers';
+
+sentryTest(
+  'should parse function identifiers that contain protocol names correctly',
+  async ({ getLocalTestPath, page, browserName }) => {
+    const url = await getLocalTestPath({ testDir: __dirname });
+
+    const eventData = await getSentryRequest(page, url);
+
+    expect(eventData.exception?.values?.[0].stacktrace?.frames).toMatchObject(
+      browserName === 'chromium'
```

Comment:
These are different assertions based on the browser type right?

Can we pull this into a helper?


Something like you give an object where the keys are functions you pass in. The keys of the object are the browser type. So if chromium run this function.

Reply:
@AbhiPrasad would a fixture for that do? -> 59239be
And if so, do you have a better name suggestion for it? 😄

Reply:
Actually hmm, maybe we just extract them out into individual functions? `runInChromium()` etc.

Reply:
That way we can also leave room to pass in options as a second argument